libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md

```diff
@@ -107,6 +137,27 @@ __Example 2__ shows how you can create a __PdfExportSettings__ object and assign
 
 {{endregion}}
 
+__Example 3__ shows how you can create a __PdfExportSettings__ object with a different set of settings. The settings used specify an OwnerPassword and UserAccessPermissions.
+
+
+#### __[C#] Example 3: Export settings__
+
+{{region cs-radpdfprocessing-formats-and-conversion-pdf-settings_1}}
+	PdfFormatProvider provider = new PdfFormatProvider();
+    PdfExportSettings settings = new PdfExportSettings();
+    settings.IsEncrypted = true;
+    settings.OwnerPassword = "0wn3rP4ssw0rd";
+    // The following permissions are exported only if the settings.IsEncrypted property is set to true
```

Comment:
This information should be also included in the description of UserAccessPermissions